packages/async-flow/src/replay-membrane.js

```diff
@@ -313,7 +320,14 @@ export const makeReplayMembrane = ({
     try {
       const hostPromise = optVerb
         ? heapVowE(hostTarget)[optVerb](...hostArgs)
-        : // @ts-expect-error XXX heapVowE
+        : // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore once we changed this from E to heapVowE,
+          // typescript started complaining that heapVowE(hostTarget)
+          // is not callable. I'm not sure if this is a just a typing bug
+          // in heapVowE or also reflects a runtime deficiency. But this
+          // case it not used yet anyway. We disable it
+          // with at-ts-ignore rather than at-ts-expect-error because
+          // the dependency-graph tests complains that the latter is unused.
```

Comment:
Answering a question from the PR description:

I think it is reasonable to make the dependency-graph tests required.

Reply:
Explained in the commit [19407b8](https://github.com/Agoric/agoric-sdk/pull/9661/commits/19407b8930ee496aaf6f8994820c6a2ec6931ad0)